launch/sim.launch

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0"?>
 <launch>
+	<param name="robot_description" textfile="$(find emc_simulator)/urdf/rosbot.urdf"/>
 	<rosparam command="load" file="$(find emc_simulator)/config/simbot_config.yaml" />
+	<node pkg="map_server" type="map_server" name="map_server" args="$(find emc_simulator)/data/heightmap_metadata.yaml"/>
+	<node pkg="robot_state_publisher" type="robot_state_publisher" name="rob_st_pub" />
 	<node pkg="emc_simulator" type="pico_simulator" name="simulator"/>
+	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find emc_simulator)/rviz/config.rviz" />
```

Comment:
Recommended to run rviz in a separate launch file. As it required to run it separately for the using the real robot, so you can reuse that launch file/alias for simulation as well.